osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -0,0 +1,356 @@
+                                                {
+                                                    new SpriteText
+                                                    {
+                                                        Anchor = Anchor.CentreLeft,
+                                                        Origin = Anchor.CentreLeft,
+                                                        Font = OsuFont.GetFont(size: 12),
+                                                        Text = HumanizerUtils.Humanize(comment.CreatedAt)
+                                                    },
+                                                    new RepliesButton(comment.RepliesCount)
+                                                    {
+                                                        Expanded = { BindTarget = childrenExpanded }
+                                                    },
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    childCommentsVisibilityContainer = new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                        Children = new Drawable[]
+                        {
+                            childCommentsContainer = new FillFlowContainer
+                            {
+                                Padding = new MarginPadding { Left = 20 },
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Vertical
+                            },
+                            deletedChildrenPlaceholder = new DeletedChildrenPlaceholder
+                            {
+                                ShowDeleted = { BindTarget = ShowDeleted }
+                            }
+                        }
+                    }
+                }
+            };
+
+            deletedChildrenPlaceholder.DeletedCount.Value = comment.GetDeletedChildrenCount;
+
+            if (comment.UserId.HasValue)
+                username.AddUserLink(comment.User);
+            else
+                username.AddText(comment.LegacyName);
+
+            if (comment.EditedAt.HasValue)
+            {
+                info.Add(new SpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Font = OsuFont.GetFont(size: 12),
+                    Text = $@"edited {HumanizerUtils.Humanize(comment.EditedAt.Value)} by {comment.EditedUser.Username}"
+                });
+            }
+
+            if (!comment.IsDeleted)
```

Comment:
This needs to support dynamic updating (based on `ShowDeleted`?)

Reply:
The goal of this pr is just a read-only access without the actual interaction with comments, but yeah, sure

Reply:
I misread this one as `if (!ShowDeleted)`... Don't worry about it.